libs/libmariadb/patches/010-link-to-libucontext.patch

```diff
@@ -0,0 +1,49 @@
+--- a/libmariadb/CMakeLists.txt
```

Comment:
Please add the description to this patch header as well, so we can see why it is added easily.

Reply:
Done